pages/edit/index.tsx

```diff
@@ -116,6 +116,19 @@ function EditForm({ baseUrl }) {
                 "There was an issue with the " +
                   "cloudflare check. Please try again.",
               );
+            } else if (
+              json.message.includes(`Member with email ${email} not found`)
+            ) {
+              return new Promise((resolve) => {
+                setTimeout(() => {
+                  setPageState(PageState.Error);
+                  setError(
+                    `Looks like the email ${email} is not associated with a ` +
+                      `Hawaiians in Tech account`,
+                  );
+                  resolve(response);
+                }, 5000); // 5-second delay if the email is not found
```

Comment:
Only tweak I'd say is that 5-seconds feels like an eternity, especially for a superficial return. I get we're trying to slow attempts down though.

As a compromise, thoughts on making it `2000`?

Reply:
updated to 2000 in commit [457c7a6](https://github.com/hawaiians/hawaiiansintech/pull/36/commits/457c7a64374b991c99047eeee5e38b4e052388fe)